src/Init/Meta.lean

```diff
@@ -1014,7 +1020,7 @@ instance : Coe (TSepArray k sep) (TSyntaxArray k) where
   coe := TSepArray.getElems
 
 instance [Coe (TSyntax k) (TSyntax k')] : Coe (TSyntaxArray k) (TSyntaxArray k') where
-  coe a := .mk a.raw
+  coe a := .mk (a.map fun s : TSyntax k => (s : TSyntax k'))
```

Comment:
```suggestion
  coe a := .mk (a.map Coe.coe)
```
A little shorter and at least as clear in this case IMO

Reply:
And we can also remove the `.mk`.